updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc

```diff
@@ -124,8 +124,8 @@ The release image signature config map allows the Cluster Version Operator (CVO)
 
 After you configure your cluster to use the locally-installed OpenShift Update Service and local mirror registry, you can use any of the following update methods:
 
-** xref:../../updating/updating-cluster-within-minor.adoc#updating-cluster-within-minor[Updating a cluster using the web console]
-** xref:../../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI]
-** xref:../../updating/preparing-eus-eus-upgrade.adoc#preparing-eus-eus-upgrade[Preparing to perform an EUS-to-EUS update]
-** xref:../../updating/update-using-custom-machine-config-pools.adoc#update-using-custom-machine-config-pools[Performing a canary rollout update]
-** xref:../../updating/updating-cluster-rhel-compute.adoc#updating-cluster-rhel-compute[Updating a cluster that includes RHEL compute machines]
\ No newline at end of file
+** xref:../../../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-within-minor[Updating a cluster using the web console]
```

Comment:
I'm not clear on why we have `#...` fragments pinning a particular anchor for these, because we seen to be linking the top-level section on each of the pages.  But `-within-minor` should no longer be a canonical part of the web-console update page, so it seems like we should either drop the fragment here or adjust to whatever the current anchor ID is for the top-level header on that page.

Same for  the following lines about maybe dropping fragments, although the `-within-minor` quibble is just about this web-console line.

Reply:
Not super sure, but I think there is some technical requirement of our publishing platform to include anchor IDs in all xref links, even if the anchor is the ID for the entire page.

As for the `within-minor`, I'll admit that the decision to refactor the filename of the web console page, but not any of the anchors within it, was somewhat arbitrary. I thought it would be worth changing the filename here, but any further refactoring would inch this current PR further into "unwieldy" territory. Plus in this PR I'm trying to limit changes to content as much as possible, but xrefs were necessary to update while anchor IDs were not.

I have other tickets in my reorg epic geared towards that sort of low level, low priority refactoring, so I'd prefer to save this sort of effort for later.

Reply:
I don't actually see that anchor in the preview page:

```console
$ curl -s https://61201--docspreview.netlify.app/openshift-enterprise/latest/updating/updating_a_cluster/updating-cluster-web-console.html | grep 'id="updating-cluster-within-minor"'
...no hits...
```

And looking at the `<h1>` header, I don't see any anchors at all:

```console
$ curl -s https://61201--docspreview.netlify.app/openshift-enterprise/latest/updating/updating_a_cluster/updating-cluster-web-console.html | grep -4 'Updating a cluster using the web console' | tail
--
      </div>
      <div class="col-xs-12 col-sm-9 col-md-9 main">
        <div class="page-header">
          <h1>
            Updating a cluster using the web console
          </h1>
        </div>
```

But also still no worries if this sort of cleanup is punted to future work.